stan/math/prim/scal/prob/beta_binomial_lcdf.hpp

```diff
@@ -103,10 +103,9 @@ namespace stan {
         const T_partials_return mu = alpha_dbl + n_dbl + 1;
         const T_partials_return nu = beta_dbl + N_dbl - n_dbl - 1;
 
-        const T_partials_return F = F32((T_partials_return)1, mu,
-                                                    -N_dbl + n_dbl + 1,
-                                                    n_dbl + 2, 1 - nu,
-                                                    (T_partials_return)1);
+        T_partials_return F;
+        F = F32((T_partials_return)1, mu, -N_dbl + n_dbl + 1, n_dbl + 2,
```

Comment:
Why is there a cast of 1?  If anything, this should be a constructor, `T_partials_return(1)`, but why does it need to be cast to the return type rather than just passed into F32?  Are the template types in F32 not independent?  If not, that's a bigger problem.